src/__tests__/components/NHSNotifyActionLink.test.tsx

```diff
@@ -0,0 +1,25 @@
+import { render, screen } from '@testing-library/react';
+import { NHSNotifyActionLink } from '@/src/components/molecules/NHSNotifyActionLink/NHSNotifyActionLink';
+import { ActionLinkType } from '@/src/components/molecules/NHSNotifyActionLink/action-link.types';
+
+const componentProps: ActionLinkType = {
+  text: 'Some text',
+  target: '#',
+};
+describe('Footer component', () => {
+  it('renders component correctly', () => {
+    render(<NHSNotifyActionLink {...componentProps} />);
+
+    expect(screen.getByTestId('action-link-wrapper')).toBeInTheDocument();
+
+    expect(screen.getByTestId('action-link')).toBeInTheDocument();
```

Comment:
do you need a separate check to see if these things are in the document if you are also checking specific properties of it later on in the test?

Reply:
Yes, there is a need to be sure the element is on the document before testing any of it's properties, if it can't find the element, then it fails earlier rather than having to look for the property of an element that does not exist